src/opera/commands/deploy.py

```diff
@@ -85,3 +77,21 @@ def deploy(args):
         return 1
 
     return 0
+
+
+def deploy(service_template: str, inputs: typing.Optional[dict], storage: Storage, num_workers: int):
+    """
+    :raises ParseError:
```

Comment:
Is this necessary? Or what does this differ from try-except?

Reply:
This doesn't replace try-except, it just documents that the function can throw these two errors. There aren't any checked exceptions in Python, so you never know what anything can throw, but this is a nice way of signalling to userss of opera as a library of what can go wrong. You can catch something better than `Exception`.

Reply:
Great, that's ok then. And thanks for the explanation.